source/backend/apimodels/Models/Concepts/Property/PropertyViewMap.cs

```diff
@@ -0,0 +1,103 @@
+using Mapster;
+using Entity = Pims.Dal.Entities;
+
+namespace Pims.Api.Models.Concepts.Property
+{
+    public class PropertyViewMap : IRegister
+    {
+        public void Register(TypeAdapterConfig config)
+        {
+            config.NewConfig<Entity.PimsPropertyLocationVw, PropertyViewModel>()
+               .Map(dest => dest.Id, src => src.PropertyId)
+               .Map(dest => dest.Pid, src => src.Pid)
+.Map(dest => dest.PidPadded, src => src.PidPadded)
```

Comment:
I will run the formater on a separate PR to address this and other issues like these.